client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -25,11 +111,80 @@ export default function PaymentMethodAdd(): ReactElement {
 				<CardHeading size={ 36 }>{ translate( 'Payment Method' ) }</CardHeading>
 			</div>
 
-			<Card className="payment-method-add__body">
-				<div className="payment-method-add__body-left"></div>
+			<CheckoutProvider
+				onPaymentComplete={ () =>
+					onPaymentSelectComplete( {
+						successCallback: () => page( '/partner-portal/payment-method/' ),
+						translate,
+						showSuccessMessage,
+					} )
+				}
+				showErrorMessage={ showErrorMessage }
+				showInfoMessage={ showInfoMessage }
+				showSuccessMessage={ showSuccessMessage }
+				paymentMethods={ paymentMethods }
```

Comment:
Note from my editor: it complains about the `paymentMethods` attribute not accepting `(PaymentMethod | null)` null values but rather `PaymentMethod[]`.

This can be traced back to `const paymentMethods = useMemo( () => [ stripeMethod ].filter( Boolean ), [ stripeMethod ] );` where it thinks `const paymentMethods` isn't a list but rather a single `PaymentMethod`.
I don't think it's the case but worth looking into 😄